ch_tools/chadmin/cli/table_group.py

```diff
@@ -363,3 +370,67 @@ def materialize_ttl_command(ctx, dry_run, all_, database, table, exclude_table):
         materialize_ttl(
             ctx, database=t["database"], table=t["table"], echo=True, dry_run=dry_run
         )
+
+
+@table_group.command("set-flag")
+@option(
+    "--database",
+    type=StringParamType(),
+    help="Filter tables to set the flag by the specified database name.",
+)
+@option(
+    "--table",
+    type=StringParamType(),
+    help="Filter in tables by the specified table name.",
+)
+@option(
+    "--exclude-table",
+    type=StringParamType(),
+    help="Filter out tables by the specified table name.",
+)
+@option("--flag", type=str, default="convert_to_replicated", help="Flag name.")
+@option(
+    "-v",
+    "--verbose",
+    type=bool,
+    is_flag=True,
+    help="Show tables and flag paths.",
+)
+@pass_context
+def set_flag_command(
+    ctx: Context,
+    database: str,
+    table: str,
+    exclude_table: str,
+    flag: str,
+    verbose: bool,
+) -> None:
+    """
+    Create a flag with the specified name inside the data directory of the table.
+    """
+    if database is None:
+        print("Database is not specified")
+        return
```

Comment:
Can be replaced with `required=True` in `@option` definition.

But I suggest to do make the check less strict: 
```
    if not any((all_, database, table)):
        ctx.fail(
            "At least one of --all, --database, --table options must be specified."
        )
```